docs/en/stack/ml/anomaly-detection/ootb-ml-jobs-apache.asciidoc

```diff
@@ -5,11 +5,88 @@
 <titleabbrev>Apache</titleabbrev>
 ++++
 // tag::apache-jobs[]
-These {anomaly-job} wizards appear in {kib} if you use 
-{filebeat-ref}/index.html[{filebeat}] to ship access logs from your 
-https://httpd.apache.org/[Apache] HTTP servers to {es} and store it using fields 
-and data types from the Elastic Common Schema (ECS). For more details, see the
-{dfeed} and job definitions in
+These {anomaly-job} wizards appear in {kib} if you use either {filebeat} or the
```

Comment:
Is it worth putting the Apache integration condition first here, as this is now the preferred way of getting these jobs.

Reply:
Sure!  I've made that change in cef48023d5c97caf1c44fbe64700d4d2378b4b50